dev/tests/static/testsuite/Magento/Test/Integrity/ComposerTest.php

```diff
@@ -73,10 +73,9 @@ public static function getBlacklist(string $pattern)
     {
         $blacklist = [];
         foreach (glob($pattern) as $list) {
-            //phpcs:ignore Magento2.Performance.ForeachArrayMerge
-            $blacklist = array_merge($blacklist, file($list, FILE_IGNORE_NEW_LINES | FILE_SKIP_EMPTY_LINES));
+            $blacklist[] = file($list, FILE_IGNORE_NEW_LINES | FILE_SKIP_EMPTY_LINES);
         }
-        return $blacklist;
+        return array_merge([], ...$blacklist);
```

Comment:
Could you replace `*****list` with less controversial word? Eg. `IgnoredList`?

Reply:
That’s not purpose of current PR. We have a lot such places in code, they need to be reviewed all and fixed at once